packages/react/src/ErrorBoundary.tsx

```diff
@@ -4,30 +4,29 @@ import { BacktraceClient } from './BacktraceClient';
 
 type RenderFallback = (error: Error) => ReactElement;
 
-export interface Props {
+export interface BacktraceErrorBoundaryProps {
     children: ReactNode;
     fallback?: ReactElement | RenderFallback;
     name?: string;
 }
 
-export interface State {
+export interface BacktraceErrorBoundaryState {
     error?: Error;
 }
 
-export class ErrorBoundary extends Component<Props, State> {
-    private _client: BacktraceClient;
+export class ErrorBoundary extends Component<BacktraceErrorBoundaryProps, BacktraceErrorBoundaryState> {
     private COMPONENT_THREAD_NAME = 'component-stack';
-    constructor(props: Props) {
+    constructor(
+        props: BacktraceErrorBoundaryProps,
+        private readonly _client: BacktraceClient = BacktraceClient.instance as BacktraceClient,
```

Comment:
I think you should not add anything outside `props` object when creating a React component.

Reply:
the constructor with the client parameter will be used by error boundaries that inherit from the react error boundary. The client parameter doesn't matter in error boundary creation, but it matters in calling base class constructor from react-native